### PR TITLE
Add support for debugger

### PR DESCRIPTION
**Pull request by obycode**
Dependent on https://github.com/hirosystems/clarity-repl/pull/116

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -457,6 +457,44 @@ $ clarinet run --allow-write https://deno.land/x/clarinet@v0.15.4/ext/stacksjs-h
...
+    w contracts/SP2KAF9RF86PVX3NEE27DFV1CQX0T4WGR41X3S45C.bitcoin-whales.payout
+    ```
+- Step execution
+  - **Step-in**: Step into the sub-expressions (`step` or `s` command)
+  - **Step-out**: Complete execution of the current expression and return the result back to the parent (`finish` or `f` command)
+  - **Step-over**: Continue to completion of the current expression, stepping over sub-expressions (`next` or `n` command)
+  - **Continue**: Continue execution until hitting a breakpoint or completing execution (`continue` or `c` command)
+- Evaluate and print the result of any expression (`print` or `p` command)
+
+To initiate a debug sessionm, first enter the REPL console using:
```
Comment (moodmosaic) [inline README.md:484]:
```suggestion
To initiate a debug session, first enter the REPL console using:
```

**Review by moodmosaic — COMMENTED** [review]
Corrected a small typo.

Very much looking forward to trying this! Thank you!

**Comment by obycode** [comment]
> Corrected a small typo.

Thanks @moodmosaic!

File: src/frontend/cli.rs [hunk 2]
```diff
@@ -462,6 +462,25 @@ pub fn main() {
                         &project_manifest.project.authors,
                     ),
                 ));
+
+                #[cfg(feature = "telemetry")]
```
Comment (lgalabru) [inline src/frontend/cli.rs:466]: Should we make sure that developers enabled telemetry in their project before?
`if manifest.project.telemetry`

**Review by lgalabru — APPROVED** [review]
🚀

File: src/frontend/cli.rs (hunk 2 above)
Comment (obycode) [reply src/frontend/cli.rs:466]: See line 457:

```rust
            if project_manifest.project.telemetry {
```

**Pull request merged**